docs/integrations/data-integrations/datastax.mdx

```diff
@@ -15,7 +15,7 @@ The required arguments to establish a connection are as follows:
 
 * `user` is the user to authenticate.
 * `password` is the password to authenticate the user.
-* `secure_connection_bundle` is the path to the `secure_connection_bundle` zip file.
+* `secure_connection_bundle` is the path to the `secure_connect_bundle` zip file.
```

Comment:
```
`secure_connect_bundle` is the path to the `secure_connect_bundle` zip file.
```
or not?

Reply:
Oh this should be named `secure_connect_bundle`. Let me rename